pkg/sqlite/tag.go

```diff
@@ -536,3 +536,63 @@ func (qb *tagQueryBuilder) GetAliases(tagID int) ([]string, error) {
 func (qb *tagQueryBuilder) UpdateAliases(tagID int, aliases []string) error {
 	return qb.aliasRepository().replace(tagID, aliases)
 }
+
+func (qb *tagQueryBuilder) MergeTags(source []int, destination int) error {
```

Comment:
This function should be moved into the `tag` package and use the Reader/Writer interfaces to perform the operation. The reader/writer interfaces are intended for low-level operations, and this function should be possible to do outside of the data layer.

It should also have an accompanying unit test.